src/write_batch.rs

```diff
@@ -77,7 +89,7 @@ unsafe extern "C" fn writebatch_delete_callback(state: *mut c_void, k: *const c_
     leaked_cb.delete(key.to_vec().into_boxed_slice());
 }
 
-impl WriteBatch {
+impl<const TRANSACTION: bool> WriteBatchWithTransaction<TRANSACTION> {
```

Comment:
const generics! i wonder it's okay to bump MSRV?

Reply:
Yeah. I think it's ok to define MSRV 1.60.0. There is PR #649 which requires MSRV as well.